app/models/heritage.rb

```diff
@@ -353,4 +354,17 @@ def apply_stack
   def delete_stack
     cf_executor.delete
   end
+
+  def validate_ssm_parameters
+    ssm_paths = environments.secrets.select{ |key|
+      key.value_from.start_with?("/barcelona/#{district.name}")
+    }.map(&:value_from)
+
+    ssm_parameter = SsmParameters.new(self.district, "")
+    response = ssm_parameter.get_parameters(ssm_paths)
+
+    if response.invalid_parameters.present?
```

Comment:
Owing to the comment above, I think we can just make a `ssm_paths_exist?(ssm_paths)` method on the `SsmParameters` class